guides/common/modules/con_mirroring-policies-overview.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Mirroring_Policies_Overview_{context}"]
+= Mirroring Policies Overview
+
+Mirroring keeps the local repository exactly in synchronization with the upstream repository. 
+If any content is removed from the upstream repository since the last synchronization, it will be removed from the local repository as well.
+
+Mirroring policies can be used for finer control over mirroring of repodata and content when synchronizing a repository. 
+For example, if it is not possible to mirror the repodata for a repository, you can set the mirroring policy to mirror only content for this repository.
+
+{ProjectServer} has the following mirroring policies:
+
+Additive::
+Neither the content nor the repodata is mirrored. Thus, only new content added since the last synchronization is added to the local repository and nothing is removed.
```

Comment:
```suggestion
Neither the content nor the repodata is mirrored.
Thus, only new content added since the last synchronization is added to the local repository and nothing is removed.
```

Reply:
Hi @maximiliankolb I think the [#1149](https://github.com/theforeman/foreman-documentation/pull/1149) is for Download policies, this one is for mirroring policies that describe how content and metadata are mirrored. Download policy will decide whether just the metadata will get downloaded with the sync or along with the content. The mirroring policy will determine how the content/metadata will be mirrored from the upstream.

Reply:
I will apply the changes that you suggested. Thanks!